src/SprykerEco/Zed/Sevensenders/Business/Api/Adapter/AdapterInterface.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
```

Comment:
This is using the wrong license. Please check with current specs on .license file for MIT.